### DISCUSSION TITLE
Accordion component

### DISCUSSION THREAD

**Discussion by buenojosias** [tasteui/tasteui#518] [Ideas]
How about an accordion or collapse component? I believe it will be very useful for developers.
An example code: [https://www.penguinui.com/components/accordion](https://www.penguinui.com/components/accordion)

**Comment by devajmeireles** [comment]
Hey, @buenojosias ! Although it seems useful, this type of component is only linked to visual aspects, without having specific functionality linked to Livewire, for example. We can do it, but I don't see it as a priority.

**Reply by buenojosias** [reply]
> Hey, @buenojosias ! Although it seems useful, this type of component is only linked to visual aspects, without having specific functionality linked to Livewire, for example. We can do it, but I don't see it as a priority.

I saw that it is really not necessary to create a component for this, since Alpine itself already provides this functionality with x-collapse. Then just use it in combination with the components already in the package.
